app/CMakeLists.txt

```diff
@@ -300,8 +300,10 @@ function(set_up_app_build)
 	endif()
 	embedded_set_target_link_includes(TARGET ${EXE_NAME} INCLUDES "${PLATFORM_LINK_INCLUDES}")
 
-	#Generate binary file from axf
+	#Generate bin/hex/elf files from axf executable
```

Comment:
For this and _all_ similar entries: i wonder if you can just add the additional info in a new line, so the diff looks more clean.

Reply:
Sure np

Reply:
Done